surf_tide/src/main.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (C) 2020 O.S. Systems Sofware LTDA
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use derive_more::From;
+use futures_util::lock::Mutex;
+use http_client::native::NativeClient;
+use std::sync::Arc;
+
+use rwcst::prelude::*;
+
+#[tokio::main]
+async fn main() {
+    let (url, _guards) = rwcst::start_remote_mock();
+    let local_client = LocalClient::new();
+    let remote_client = RemoteClient::new(&url);
+    let app = App::new(remote_client);
+    rwcst::run(local_client, app).await;
+}
+
+struct LocalClient {
+    client: surf::Client<NativeClient>,
+}
+
+struct RemoteClient {
+    client: surf::Client<NativeClient>,
+    remote: String,
+}
+
+struct App {
+    info: Arc<Mutex<rwcst::Info>>,
+    client: RemoteClient,
+}
+
+#[derive(Debug, From)]
+enum Err {
+    Server(tide::Error),
+    //Client(surf::Error),
```

Comment:
Is this variant never constructed? If so we can drop the comment line. But how is not used? What error does the client return?